tests/torch/nas/test_search_space.py

```diff
@@ -93,7 +93,7 @@ def __str__(self):
         ElasticityDim.DEPTH,
         model_cls=DepthBasicConvTestModel,
         params={"min_block_size": 1, "hw_fused_ops": False},
-        ref_search_space=[[]],
+        ref_search_space=[[0], []],
```

Comment:
After modifying the original padding value of `DepthBasicConvTestModel`, some output shape are different from the one in the original old model. So elastic depth has a larger search space.